components/tyk-gateway/templates/deployment-gw-repset.yaml

```diff
@@ -143,9 +143,27 @@ spec:
           - name: TYK_GW_ENABLENONTRANSACTIONALRATELIMITER
             value: "true"
           - name: TYK_GW_POLICIES_POLICYSOURCE
+        {{- if .Values.global.isDataplane }}
+            value: "rpc"
+        {{- else}}
             value: "file"
+        {{- end}}
+        {{- if .Values.global.isDataplane }}
+          - name: TYK_GW_ANALYTICSCONFIG_TYPE
+            value: "rpc"
+          - name: TYK_GW_SLAVEOPTIONS_USERPC
+            value: "true"
+          - name: TYK_GW_AUTHOVERRIDE_FORCEAUTHPROVIDER
+            value: "true"
+          - name: TYK_GW_AUTHOVERRIDE_AUTHPROVIDER_STORAGEENGINE
```

Comment:
I can't see this environment variable in the documentation. Can you double check if it is correct?